### PR TITLE
add geolocation

### PR DESCRIPTION
**Pull request by thiloSchlemmer**
Adding geolocation functionality

### CODE REVIEW

File: src/modules/map/components/geolocationButton/assets/geo-alt-fill.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-geo-alt-fill" viewBox="0 0 16 16">
```
Comment (taulinger) [inline src/modules/map/components/geolocationButton/assets/geo-alt-fill.svg:1]: License is missing

File: src/modules/map/components/geolocationButton/geolocationButton.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
+.geolocation {
+  position: absolute;
+  right: 10px;
+  top: 50px;
```
Comment (taulinger) [inline src/modules/map/components/geolocationButton/geolocationButton.css:4]: for the moment  at top: `300px` ?

File: src/modules/map/components/geolocationButton/geolocationButton.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+.geolocation-button {
+  background-color: var(--map-button-bg-color);
+  display: inline-block;
+  height: 40px;
+  width: 40px;
+  border-radius: 50%;
+  border: none;
+  outline: none;
+  padding: 5px;
```
Comment (taulinger) [inline src/modules/map/components/geolocationButton/geolocationButton.css:42]: `padding-top: 4px` would be enough

**Comment by taulinger** [comment]
After that, the time has come to convert it to a PR and do the review...

**Comment by taulinger** [comment]
We should not forget to remove the geolocation buttons in the show room ...

File: src/modules/map/components/geolocationButton/geolocationButton.css (hunk 2 above)
Comment (thiloSchlemmer) [reply src/modules/map/components/geolocationButton/geolocationButton.css:4]: ok

File: src/modules/map/components/geolocationButton/geolocationButton.css (hunk 3 above)
Comment (thiloSchlemmer) [reply src/modules/map/components/geolocationButton/geolocationButton.css:42]: ok

**Review by taulinger — APPROVED** [review]
Harder than thought, but finally done 🥳

**Pull request merged**